models/DataObject/ClassDefinition/Data/CalculatedValue.php

```diff
@@ -194,7 +194,7 @@ public function getDataFromEditmode($data, $object = null, $params = [])
      * @param DataObject\Concrete|null $object
      * @param mixed $params
      *
-     * @return float
+     * @return string
```

Comment:
is there something wrong with $data then? returns it 1:1

Reply:
The parent method has `return string` So all children methods should also be `return string`. I can change it to `return (string)$data;`

Reply:
go for it

Reply:
I use the getDataForEditmode method to get the correct string